packages/emotion/native/Css.ml

```diff
@@ -335,3 +336,36 @@ let render_style_tag () =
            Printf.sprintf "%s %s" accumulator rules)
        ""
   |> String.trim
+
+let get_style_rules = render_style_tag
+
+let render_style_html_tag () =
+  Stylesheet.get_all instance
+  |> List.fold_left
+       (fun (prevDataEmotion, prevStyles) (hash, rules) ->
+         match rules with
+         | Classnames rules ->
+           let rules =
+             pp_rules hash rules
+             |> Printf.sprintf "%s %s" prevStyles
+             |> String.trim
+           in
+           let dataEmotion =
+             Printf.sprintf "%s %s" prevDataEmotion hash |> String.trim
+           in
+           dataEmotion, rules
+         | Keyframes keyframes ->
+           let rules =
+             pp_keyframes hash keyframes
+             |> Printf.sprintf "%s %s" prevStyles
+             |> String.trim
+           in
+           let dataEmotion =
+             Printf.sprintf "%s %s" prevDataEmotion hash |> String.trim
+           in
+           dataEmotion, rules)
+       ("css", "")
+  |> (fun (dataEmotion, styles) ->
```

Comment:
Can you use render_style_tag inside render_style_html_tag?

Reply:
Hey @davesnx , to use the `render_style_tag` we would need something like this:

```ocaml
(** Deprecated: Use get_string_style_rules instead*)
let render_style_tag () =
  Stylesheet.get_all instance
  |> List.fold_left
       (fun accumulator (hash, rules) ->
         match rules with
         | Classnames rules ->
           let rules = pp_rules hash rules |> String.trim in
           Printf.sprintf "%s %s" accumulator rules
         | Keyframes keyframes ->
           let rules = pp_keyframes hash keyframes |> String.trim in
           Printf.sprintf "%s %s" accumulator rules)
       ""
  |> String.trim

let get_string_style_rules = render_style_tag

let get_string_style_hashes () =
  Stylesheet.get_all instance
  |> List.fold_left
       (fun accumulator (hash, _) ->
         Printf.sprintf "%s %s" accumulator hash |> String.trim)
       ""

let render_style_html_tag () =
  Printf.sprintf "<style data-s data-emotion=\"css %s\">%s</style>"
    (get_string_style_hashes ()) (get_string_style_rules ())

```
WDYT?

I'm also trying to define a better name for those functions, any idea about it?